_config.yml

```diff
@@ -118,11 +118,16 @@ defaults:
       type: security_updates
     values:
       layout: styleguide
+  - scope:
+      path: ""
+    values:
```

Comment:
What are these empty values for?

Reply:
From my understanding, you use these values to define the defaults. [^1] So here it's setting the default image for link sharing to the black logo but can be overwritten by the image being declared on another page like in the example I commented above where I set the image to `/img/uswds-logo/gsa-logo.svg`.

Technically, `values` is not empty because `image:` is actually nested beneath it.

Most other defaults within `_config.yml` actually have the same set up.

https://github.com/uswds/uswds-site/blob/74788f7aa9849be56a06f86c3b8085fd9c0050fd/_config.yml#L81-L97

I tried setting the image without using this format and it didn't work

[^1]: https://jekyllrb.com/docs/configuration/front-matter-defaults/

Reply:
Makes sense, thank you!